test/only.js

```diff
@@ -1,25 +0,0 @@
-QUnit.module( "QUnit.only", function( hooks ) {
```

Comment:
This deleted file is causing a build failure (I see the 404's print in my local dev, but it didn't indicate a failure like Travis did).
It was being used in the "sandboxed-iframe" test, I _think_ to try to isolate it as much as possible. That should be resolved now with the CLI test suite, and I think some or all of the "bridge" injections can go away now. I'm still grasping at the intent of that sandbox test, so I'm curious of any historical knowledge others can share.

Reply:
Based on the commit history for these files, I believe its purpose is not so much testing of the `QUnit.only` functionality, but rather it is a higher-level integration test for running QUnit from inside an iframe. Specifically, a sandbox iframe that has limited capabilitities. Ref https://github.com/qunitjs/qunit/pull/1091.

As such, some form of this should remain. See https://github.com/qunitjs/qunit/pull/1091#issuecomment-274123666 for more information. The test needs to cause use of code that tries to use `QUnit.config.storage` (aka localSessionStorage, aka window.sessionStorage).

Reply:
Based on the commit history for these files, I believe its purpose is not so much testing of the `QUnit.only` functionality, but rather it is a higher-level integration test for running QUnit from inside an iframe. Specifically, a sandbox iframe that has limited capabilitities. Ref https://github.com/qunitjs/qunit/pull/1091.

As such, some form of this should remain. See https://github.com/qunitjs/qunit/pull/1091#issuecomment-274123666 for more information. The test needs to trigger code that tries to use `QUnit.config.storage` (aka localSessionStorage, aka window.sessionStorage).

Reply:
I've restored this sandboxed-iframe test.
I brought back the only.js test logic, albeit renamed to `sandboxed-iframe.js` to not be confused with the `only` tests.
(also, I now see some whitespace changes in the HTMLs that I can correct right away)

It sounds like this might be able to be improved, but this is admittedly beyond my scope, so I'm restoring to status quo to keep issues independent.